src/js/utils/dom.js

```diff
@@ -215,5 +216,11 @@ export function openLink(link, target, additionalOptions = {}) {
     a.href = link;
     a.target = target;
     a = Object.assign(a, additionalOptions);
-    a.dispatchEvent(new MouseEvent(`click`, { bubbles: true, cancelable: true, view: window }));
+
+    // Must support all browsers.
```

Comment:
> // Must support all browsers.
That kind of goes without saying.

Why only create an event in `firefox`? I thought we wanted to do that in all modern browsers, and just call click in `ie`.

Reply:
Firefox is the only modern browser that doesn't support clicking orphaned anchors.

Reply:
Yes, but we don't need a comment that says "Must support all browsers".